experiment/autobumper/bumper/bumper.go

```diff
@@ -39,13 +41,39 @@ const (
 	testImageRepo   = prowRepo
 )
 
-func Call(cmd string, args ...string) error {
+func Call(stdout, stderr io.Writer, cmd string, args ...string) error {
 	c := exec.Command(cmd, args...)
-	c.Stdout = os.Stdout
-	c.Stderr = os.Stderr
+	c.Stdout = stdout
+	c.Stderr = stderr
 	return c.Run()
 }
 
+const (
+	censored = "CENSORED"
+)
+
+var censoredBytes = []byte(censored)
+
+type HideSecretsWriter struct {
+	SecretsMap map[string][]byte
+	Delegate   io.Writer
+}
+
+func (w HideSecretsWriter) Write(content []byte) (int, error) {
+	l := len(content)
+	var newContent = make([]byte, l)
+	_ = copy(newContent, content)
+	for sKey := range w.SecretsMap {
```

Comment:
We should ask for the `secretAgent` and use it's method `Censor()`

Reply:
Much cleaner now.